src/main/java/org/openrewrite/staticanalysis/CatchClauseOnlyRethrows.java

```diff
@@ -113,6 +115,13 @@ private boolean onlyRethrows(J.Try.Catch aCatch) {
                 }
 
                 Expression exception = ((J.Throw) aCatch.getBody().getStatements().get(0)).getException();
+
+                // In C# an implicit rethrow is possible
+                if (getCursor().firstEnclosing(SourceFile.class) instanceof Cs &&
+                    exception instanceof J.Empty) {
```

Comment:
How certain are we that this is always modeled as a `J.Empty`, and not as let's say `null`?

Reply:
Mostly asking since elsewhere we had problems when recipes were chained; so by and of themselves they'd be fine, but when chained together a previous recipe might `return null`. Although thinking about it; this would then just not match and make no changes presumably; that's fine.